<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Магазин юнитов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QListWidget" name="teamListWidget"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="deactivatePushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Деактивировать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Купленные юниты</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="buyPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Купить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QListWidget" name="shopListWidget"/>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="availabilityListWidget"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="activatePushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Активировать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Магазин доступных персонажей</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Текущая команда</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="backPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="budgetLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Бюджет: 0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="levelLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Уровень: 1 (0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Система</string>
    </property>
    <addaction name="settingsAppAction"/>
    <addaction name="backAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Аккаунт</string>
    </property>
    <addaction name="aboutAccountAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAppAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutAppAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="aboutAccountAction">
   <property name="text">
    <string>Информация об аккаунте</string>
   </property>
  </action>
  <action name="backAction">
   <property name="text">
    <string>Назад</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Закрыть приложение</string>
   </property>
  </action>
  <action name="settingsAppAction">
   <property name="text">
    <string>Настройка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
